config/jobs/kubernetes/sig-autoscaling/azure/cluster-autoscaler-azure-presubmits.yaml

```diff
@@ -42,7 +42,7 @@ presubmits:
           - name: ADDITIONAL_ASO_CRDS
             value: authorization.azure.com/*;managedidentity.azure.com/* # needed for this CLUSTER_TEMPLATE
           - name: KUBERNETES_VERSION
-            value: "1.30" # latest available in AKS
+            value: "1.32" # latest available in AKS
```

Comment:
Should we also add a job for the 1.32 cluster-autoscaler branch?

I also see the 1.31 job here is using AKS 1.30, so that should be updated too sometime.

Reply:
yep, will address